docs/admin/federation/index.md

```diff
@@ -137,11 +137,11 @@ cluster.
 Assuming our YAML file is located at `/cluster1/cluster.yaml`, we can run the following command to register this cluster:
 
 ```shell
-$ kubectl create -f /cluster1/cluster.yaml --cluster=federation-cluster
+$ kubectl create -f /cluster1/cluster.yaml --context=federation-cluster
```

Comment:
OK, I'd suggest that you add a TODO, and associated issue to fix that.  It seems like that's probably where the original confusion originated (i.e. Kelsey using `--cluster` instead of `--context`, due to the thing being named `federation-cluster`).  Very confusing to the user, I would imagine.